src/node_api.cc

```diff
@@ -114,11 +114,11 @@ void node_napi_env__::EnqueueFinalizer(v8impl::RefTracker* finalizer) {
 
 void node_napi_env__::DrainFinalizerQueue() {
   // As userland code can delete additional references in one finalizer,
-  // the list of pending finalizers may be mutated as we execute them, so
-  // we keep iterating it until it is empty.
+  // the list may grow while draining, so keep iterating until empty.
```

Comment:
Would you mind keeping the original comment that "the list may be mutated", instead of it may only grow? Items may also be removed from the list during finalization.

Reply:
sure, thanks